web-api/tests/document_candidates.rs

```diff
@@ -181,3 +181,50 @@ async fn test_candidates_warning() {
     })
     .await;
 }
+
+#[tokio::test]
+async fn test_candidates_reingestion() {
```

Comment:
:art: this is quite similar to the test in `tests/ingestion.rs` l.135. i didn't take a full look, is this a duplication maybe?